packages/api-v2/src/main.ts

```diff
@@ -9,6 +9,9 @@ async function bootstrap() {
     logger: ["log", "error", "warn", "debug", "verbose"],
   });
 
+  // Enabled for testing :(
+  app.enableCors();
```

Comment:
Enabled CORS for testing, but as I indicated earlier, this won't be needed for prod as both front and back end will both be on the same URL. Thus no requirement for CORS.